src/pages/wallet/send.vue

```diff
@@ -271,6 +282,76 @@ export default {
       this.newTx.address = info.address;
       this.newTx.payment_id = info.payment_id;
     },
+    // helper for constructing a dialog for confirming transactions
+    confirmTransactionDialog(val) {
+      // the split_transfer rpc call can return several txs
+      const { feeList, amountList, destinations, metadataList, priority } = val.txData;
+      const totalFees = feeList.reduce((a, b) => a + b, 0) / 1e9;
+      const totalAmount = amountList.reduce((a, b) => a + b, 0) / 1e9;
+      // a tx can be split, but only sent to one address
+      const destination = destinations[0].address;
+      const note = this.newTx.note;
+      // not stored on chain, but we should only save a note to a completed tx
+      const isBlink = [0, 2, 3, 4, 5].includes(priority) ? true : false;
+      const blinkOrSlow = isBlink ? "Blink" : "Slow";
+      const { name, description, save } = this.newTx.address_book;
+      const addressSave = {
+        address: this.newTx.address,
+        payment_id: this.newTx.payment_id,
+        address_book: {
+          description,
+          name,
+          save
+        }
+      };
+      const message = `
+  <div class="confirm-tx-dialog">
+  <span class="label">Sending to:</span> ${destination}<br /><br />
+  <span class="label">Amount:</span> ${totalAmount} Loki<br />
+  <span class="label">Total fees:</span> ${totalFees} Loki<br />
+  <span class="label">Send speed:</span> ${blinkOrSlow}
+  </div>
+  `;
+
+      const relayTxData = {
+        metadataList,
+        isBlink,
+        addressSave,
+        note
+      };
+      this.$q
```

Comment:
There is also an alternative way to do this: https://quasar.dev/vue-components/dialog

Essentially the dialog html is triggered on a flag instead of using a function.
We should move to using it like described above in the future for complex dialogs.

Reply:
yeah I can't remember why I decided to do it this way, but I did think about that way